pkgs/applications/misc/tellico/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, fetchurl
+, mkDerivation
+, kdeApplications
+, kinit
+, kdelibs4support
+, solid
+, kxmlgui
+, karchive
+, kfilemetadata
+, khtml
+, knewstuff
+, libksane
+, cmake
+, exempi
+, extra-cmake-modules
+, libcdio
+, poppler
+, makeWrapper
+, kdoctools
+}:
+
+mkDerivation rec {
+  name = "tellico";
+  version = "3.3.0";
+  pversion = "3.3";
```

Comment:
hmm, this probably not be picked up by the automatic updates. 
maybe something like `"${lib.versions.major version}.${lib.versions.minor version}"` would be better?

Reply:
oh even better, there is `lib.versions.majorMinor`